Formula/flarectl.rb

```diff
@@ -0,0 +1,28 @@
+class Flarectl < Formula
+  desc "CLI application for interacting with a Cloudflare account"
+  homepage "https://github.com/cloudflare/cloudflare-go/tree/master/cmd/flarectl"
+  url "https://github.com/cloudflare/cloudflare-go/archive/v0.11.6.tar.gz"
+  sha256 "def3f0b64da5a473a942303d297ccab072044856ca023fe60a435e4145aadcfd"
+  head "https://github.com/cloudflare/cloudflare-go.git"
+
+  depends_on "go" => :build
+  depends_on "gox" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
```

Comment:
AFAIK GOPATH is deprecated and should not be used anymore.

Reply:
This was following instructions from upstream.  I did just looks and looks like the instructions have been updated, so I'll see if I can build with the new instructions.  (No reliance on GOPATH or gox.)

Reply:
No longer setting GOPATH, just a simple cd and `go build`.